db/models.py

```diff
@@ -17,7 +22,7 @@ def __str__(self) -> str:
 
 
 class Movie(models.Model):
-    title = models.CharField(max_length=255)
+    title = models.CharField(max_length=255, db_index=True)
```

Comment:
`title` should not be a db index, there is a possibility that different films will have the same title. Remove this change from a model here.